front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/HomepageBanner/index.tsx

```diff
@@ -85,17 +84,39 @@ type ErrorType = 'banner_cta_signed_out_url' | 'banner_cta_signed_in_url';
 
 type Props = {
   homepageSettings: IHomepageSettingsAttributes;
+  image?: {
+    dataCode?: string;
+    imageUrl?: string;
+  };
   hasError?: boolean;
   errors?: ErrorType[];
 };
 
-const HomepageBanner = ({ homepageSettings }: Props) => {
+const HomepageBanner = ({ homepageSettings, image }: Props) => {
   const [search] = useSearchParams();
 
   return search.get('variant') === 'signedIn' ? (
-    <SignedInHeader homepageSettings={homepageSettings} />
+    <SignedInHeader
+      homepageSettings={{
+        ...homepageSettings,
+        header_bg: {
```

Comment:
We keep the structure of the component props the same while changing the way the data is stored in the craftjs json.